main.py

```diff
@@ -0,0 +1,14 @@
+import os
+from src import Glykon
+
+if __name__ == "__main__":
+
+    if ".env" not in os.listdir():
+
+        print("⇒ Running configurations")
+
+        token = input("🔴 paste your bot's token 🔴:")
```

Comment:
I would prefer an error instead, letting the user know that the config is invalid